lib/std/time.zig

```diff
@@ -51,25 +85,28 @@ pub fn milliTimestamp() u64 {
         const err = os.wasi.clock_time_get(os.wasi.CLOCK_REALTIME, 1, &ns);
         assert(err == os.wasi.ESUCCESS);
 
-        const ns_per_ms = 1000;
-        return @divFloor(ns, ns_per_ms);
+        return ns;
     }
     if (comptime std.Target.current.isDarwin()) {
-        var tv: os.darwin.timeval = undefined;
-        var err = os.darwin.gettimeofday(&tv, null);
-        assert(err == 0);
-        const sec_ms = tv.tv_sec * ms_per_s;
-        const usec_ms = @divFloor(tv.tv_usec, us_per_s / ms_per_s);
-        return @intCast(u64, sec_ms + usec_ms);
+        // https://stackoverflow.com/a/21352348
```

Comment:
I'm confused why we need this at all. 
OSX 10.13+ has a working `CLOCK_REALTIME`. 
This code is only required to support <= OSX 10.12 (which is unsupported by zig in general).

I guess we can keep it for older OSX, in which case we should put it behind a version check:
```zig
if (switch(std.Target.current.os.tag) {
    .macosx => std.Target.current.os.version_range.semver.min.major == 10 and std.Target.current.os.version_range.semver.min.minor < 12,
    // TODO: figure out which version CLOCK_REALTIME was added,
    .ios, .watchos, .tvos => false,
    else => false,
}) {
```